### PR TITLE
Rake task to compare links with expanded links

### PR DESCRIPTION
**Pull request by dougdroper**
Uses HashDiff to print out the differences of links and expanded_links
to ensure there are no breaking changes

### CODE REVIEW

File: lib/tasks/compare.rake [hunk 1]
```diff
@@ -0,0 +1,24 @@
+require 'compare_links'
+
+namespace :compare do
+  desc """
+  Compare expanded links with links for migration for a schema_name
+  rake 'compare:links[service_manual_guide]'
+  """
+  task :schema_name, [:schema_name] => :environment do |_t, args|
+    schema_name = args[:schema_name]
+    ContentItem.where(schema_name: schema_name).each do |item|
```
Comment (danielroseman) [inline lib/tasks/compare.rake]: quibble but this could be thousands of items, so probably better to use `find_each`.

**Comment by danielroseman** [comment]
One minor comment otherwise 👍

File: lib/tasks/compare.rake (hunk 1 above)
Comment (dougdroper) [inline lib/tasks/compare.rake]: There isn't a `find_each` on this ContentItem. It isn't an `ActiveRecord`object

Comment (danielroseman) [inline lib/tasks/compare.rake]: Ah yes, just ignore me...

**Pull request merged**